post_installation_configuration/machine-configuration-tasks.adoc

```diff
@@ -12,6 +12,13 @@ Aside from a few specialized features, most changes to operating systems on {pro
 
 Tasks in this section describe how to use features of the Machine Config Operator to configure operating system features on {product-title} nodes.
 
+[IMPORTANT]
+====
+NetworkManager stores new network configurations to /etc/NetworkManager/system-connections/ in a key-file format
```

Comment:
I am not 100% on "key-file" I was looking at what the ISG says about hyphens and I think it is okay based on what I see about [compound words](https://www.ibm.com/docs/en/ibm-style?topic=punctuation-hyphens#compound-words) but it does say to "Consider using a prepositional phrase after a noun instead of a compound adjective but I will leave that up to you.

Reply:
Let me remove the hyphen.